micromasters/admin.py

```diff
@@ -0,0 +1,7 @@
+"""
+Admin for the micromasters app
+"""
+
+from rest_framework.authtoken.admin import TokenAdmin
+
+TokenAdmin.raw_id_fields = ('user',)
```

Comment:
It is in the documentation about how to install the Admin for the tokens.
Not sure about the details.